CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+Unreleased
```

Comment:
@jleclanche Do we need the changelog entry?  It looks like they are reserved for releases.